onert-micro/examples/TizenRT/Makefile

```diff
@@ -0,0 +1,341 @@
+###########################################################################
+#
+# Copyright 2024 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
+-include $(TOPDIR)/.config
+-include $(TOPDIR)/Make.defs
+
+OBJEXT		?= .o
+ASRCS		=
+CXXSRCS		=
+
+CXXFLAGS += -Wno-shadow -Wno-pedantic -fno-permissive
+CXXFLAGS += -pipe -std=c++14
+CXXFLAGS += -fno-exceptions -fcheck-new -fno-rtti
+CXXFLAGS += -ffunction-sections -fdata-sections
+CXXFLAGS += -Os
+# for using quantized models disable this flag
+CXXFLAGS += -DDIS_QUANT
+CXXFLAGS += -Wno-maybe-uninitialized
+CXXFLAGS += -Wno-missing-field-initializers
+CXXFLAGS += -Wno-type-limits -Wno-undef
+
+CFLAGS += -Wno-strict-prototypes
+
+ONERTMICRO_SRC_DIR = $(TOPDIR)/../external/onert-micro/onert-micro/onert-micro/src
+ONERTMICRO_INCLUDE_DIR = $(TOPDIR)/../external/onert-micro/onert-micro/onert-micro/include
+ONERTMICRO_PAL_MCU_DIR = $(TOPDIR)/../external/onert-micro/onert-micro/onert-micro/include/pal/mcu
+ONERTMICRO_PAL_CMSISNN_DIR = $(TOPDIR)/../external/onert-micro/onert-micro/onert-micro/include/pal/cmsisnn
+ONERTMICRO_PAL_COMMON_DIR = $(TOPDIR)/../external/onert-micro/onert-micro/onert-micro/include/pal/common
+FLATBUFFER_DIR = $(TOPDIR)/../external/onert-micro
+SCHEMA_DIR = $(TOPDIR)/../external/onert-micro/externals/gen
+
+CXXFLAGS += -I$(SCHEMA_DIR) -I$(ONERTMICRO_INCLUDE_DIR) -I$(ONERTMICRO_SRC_DIR) -I$(FLATBUFFER_DIR)
+CXXFLAGS += -I$(ONERTMICRO_PAL_COMMON_DIR)
+
+ifeq ($(CONFIG_EXTERNAL_CMSIS_NN), y)
+CXXFLAGS += -I$(ONERTMICRO_PAL_CMSISNN_DIR)
+else
+CXXFLAGS += -I$(ONERTMICRO_PAL_MCU_DIR)
+endif
+ 
+
```

Comment:
Is it important to use additional line here?

Reply:
changed, thank you!